spec/system/ai_bot/persona_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+RSpec.describe "AI personas", type: :system, js: true do
+  fab!(:admin)
+
+  before do
+    SiteSetting.ai_bot_enabled = true
+    SiteSetting.ai_bot_enabled_chat_bots = "gpt-4|gpt-3.5-turbo"
+    sign_in(admin)
+  end
+
+  it "allows creation of a persona" do
+    visit "/admin/plugins/discourse-ai/ai_personas"
+    find(".ai-persona-list-editor__header .btn-primary").click()
+    find(".ai-persona-editor__name").set("Test Persona")
+    find(".ai-persona-editor__description").fill_in(with: "I am a test persona")
+    find(".ai-persona-editor__system_prompt").fill_in(with: "You are a helpful bot")
+    find(".ai-persona-editor__save").click()
+
+    wait_for(timeout: 2) { !page.current_path.include?("new") }
```

Comment:
You don't need this, you can do:

```
  expect(page).not_to have_current_path("/admin/plugins/discourse-ai/ai_personas/new")
```

Reply:
oh this will time out for me, nice

Reply:
Yep generally if you are manually waiting for something in capybara, you are probably doing the wrong thing, and there is probably a better way.